scenes/song_selection/song_selection_item.tscn

```diff
@@ -5,7 +5,7 @@
 [ext_resource type="FontFile" uid="uid://df06bw7a1s1u7" path="res://fonts/Montserrat-Bold.ttf" id="3_lp4ge"]
 [ext_resource type="FontFile" uid="uid://cl8syiom7jxno" path="res://fonts/Montserrat-Regular.ttf" id="4_sgiw1"]
```

Comment:
Spotted these fonts, which are licensed under the OFL. Make sure we have the license included with the app as that's a requirement. See https://fonts.google.com/specimen/Montserrat/about and https://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web.

Reply:
looks safe to use